modules/bootkube/resources/bootkube.sh

```diff
@@ -20,6 +20,19 @@ then
 	cp --recursive kco-bootstrap/manifests .
 fi
 
+if [ ! -d kube-apiserver-bootstrap ]
+then
+	echo "Rendering Kubernetes API server core manifests..."
+
+	# shellcheck disable=SC2154
+	podman run \
+		--volume "$PWD:/assets:z" \
```

Comment:
this assumes the $PWD has the generated/tls secrets, is that assumption correct?